src/fides/api/alembic/migrations/versions/32497f08e227_translation_cleanup.py

```diff
@@ -0,0 +1,380 @@
+        "ix_privacyexperienceconfig_banner_enabled",
+        table_name="privacyexperienceconfig",
+    )
+    op.drop_column("privacyexperienceconfig", "description")
+    op.drop_column("privacyexperienceconfig", "banner_enabled")
+    op.drop_column("privacyexperienceconfig", "title")
+    op.drop_column("privacyexperienceconfig", "is_default")
+    op.drop_column("privacyexperienceconfig", "accept_button_label")
+    op.drop_column("privacyexperienceconfig", "version")
+    op.drop_column("privacyexperienceconfig", "reject_button_label")
+    op.drop_column("privacyexperienceconfig", "banner_title")
+    op.drop_column("privacyexperienceconfig", "privacy_preferences_link_label")
+    op.drop_column("privacyexperienceconfig", "privacy_policy_link_label")
+    op.drop_column("privacyexperienceconfig", "privacy_policy_url")
+    op.drop_column("privacyexperienceconfig", "acknowledge_button_label")
+    op.drop_column("privacyexperienceconfig", "banner_description")
+    op.drop_column("privacyexperienceconfig", "save_button_label")
+    op.drop_index(
+        "ix_privacyexperienceconfighistory_banner_enabled",
+        table_name="privacyexperienceconfighistory",
+    )
+    op.drop_column("privacyexperienceconfighistory", "banner_enabled")
+    op.drop_column("privacyexperienceconfighistory", "experience_config_id")
+    op.drop_index("ix_privacynotice_regions", table_name="privacynotice")
+    op.drop_column("privacynotice", "description")
+    op.drop_column("privacynotice", "version")
+    op.drop_column("privacynotice", "displayed_in_privacy_center")
+    op.drop_column("privacynotice", "regions")
+    op.drop_column("privacynotice", "displayed_in_overlay")
+    op.drop_column("privacynotice", "displayed_in_api")
+    op.drop_index("ix_privacynoticehistory_regions", table_name="privacynoticehistory")
+    op.drop_constraint(
+        "privacynoticehistory_privacy_notice_id_fkey",
+        "privacynoticehistory",
+        type_="foreignkey",
+    )
+    op.drop_column("privacynoticehistory", "privacy_notice_id")
+    op.drop_column("privacynoticehistory", "displayed_in_privacy_center")
+    op.drop_column("privacynoticehistory", "displayed_in_overlay")
+    op.drop_column("privacynoticehistory", "regions")
+    op.drop_column("privacynoticehistory", "displayed_in_api")
+    op.drop_index(
+        "ix_privacynoticetemplate_regions", table_name="privacynoticetemplate"
+    )
+    op.drop_column("privacynoticetemplate", "description")
+    op.drop_column("privacynoticetemplate", "displayed_in_privacy_center")
+    op.drop_column("privacynoticetemplate", "regions")
+    op.drop_column("privacynoticetemplate", "displayed_in_overlay")
+    op.drop_column("privacynoticetemplate", "displayed_in_api")
+
+    # Resetting PrivacyExperienceConfigHistory language back to non-nullable after data migration
+    op.alter_column(
+        "privacyexperienceconfighistory",
+        "language",
+        existing_type=sa.VARCHAR(),
+        nullable=False,
```

Comment:
I am second guessing having the data migration backfill as "english" might not be accurate.  Instead thinking that we have this column be nullable, and don't backfill historical records and just collect going forward.